### PR TITLE
Updating Metrics 4 and 6 to use permutation test

### PR DESCRIPTION
**Pull request by ryanzhang22**

### CODE REVIEW

File: Gender Analysis.ipynb [hunk 1]
```diff
@@ -1273,19 +1274,14 @@
...
     "\n",
-    "        np.random.shuffle(projected_features_scenes)\n",
-    "        clf_random.fit(projected_features_scenes, labels)\n",
-    "        acc_random = clf_random.score(projected_features_scenes, labels)\n",
-    "        value = acc / acc_random\n",
+    "        base_acc, rand_acc, p_value = permutation_test_score(clf, projected_features_scenes, labels, scoring=\"accuracy\", n_permutations=100)\n",
+    "        rand_acc = np.mean(rand_acc)\n",
     "        \n",
-    "        if value <= 1.2 and acc <= .7 and len(labels) < 25: # can tune as desired\n",
+    "        if p_value > 0.1: # can tune as desired\n",
```
Comment (ryanzhang22) [inline Gender Analysis.ipynb]: This can be updated if needed

File: Geography Analysis.ipynb [hunk 2]
```diff
@@ -1553,9 +1554,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.3"
+   "version": "3.7.3-final"
```
Comment (Angelina-Wang) [inline Geography Analysis.ipynb]: please remove change

File: Gender Analysis.ipynb (hunk 1 above)
Comment (Angelina-Wang) [reply Gender Analysis.ipynb]: besides just p-value, can there be a test for strength of association as well? i.e., if p-value is very low but distributions are very different, that's not as important

Comment (Angelina-Wang) [reply Gender Analysis.ipynb]: discussed over slack, but let's say if p_value < .05, and then based on your qualitative inspection, feel free to pick some acc or strength cut off that seems reasonable to go here

**Pull request merged**